digitalocean/config.go

```diff
@@ -17,29 +20,57 @@ import (
 )
 
 type Config struct {
-	Token string
+	Token     string
+	AccessID  string
+	SecretKey string
+}
+
+type CombinedConfig struct {
+	client    *godo.Client
+	accessID  string
+	secretKey string
+}
+
+func (c *CombinedConfig) godoClient() *godo.Client { return c.client }
+
+func (c *CombinedConfig) awsClient(region string) (*session.Session, error) {
+	endpoint := fmt.Sprintf("https://%s.digitaloceanspaces.com", region)
+	client, err := session.NewSession(&aws.Config{
+		Region:      aws.String("us-east-1"),
```

Comment:
I cant remember off the top of my head how the aws golang library works but should this value be hard carded to us-east-1?

Reply:
Well, I can't find the reference anymore but I seem to remember this being a dummy value since the  Digital Ocean region is specified in the endpoint URL.  This was months ago so it is entirely possible that this was the case at one point but not now.  I'll keep looking into it until I find something definitive.